Themes/default/scripts/jquery.sceditor.smf.js

```diff
@@ -209,25 +209,19 @@ sceditor.command.set(
 			var editor = this;
 
 			editor.commands.link._dropDown(editor, caller, function (url, text) {
-				// needed for IE to restore the last range
-				editor.focus();
-
-				// If there is no selected text then must set the URL as
-				// the text. Most browsers do this automatically, sadly
-				// IE doesn't.
 				if (!editor.getRangeHelper().selectedHtml() || text) {
 					text = text || url;
 
 					editor.wysiwygEditorInsertHtml(
-						'<a target="_blank" rel="noopener" href="' +
+						'<a data-type="url" href="' +
```

Comment:
```suggestion
						'<a data-type="url" target="_blank" rel="noopener" href="' +
```

Reply:
Are you sure? SCEditor transforms `target` into `data-sce-target`